src/neuroglancer/trackable_value.ts

```diff
@@ -64,6 +64,13 @@ export class TrackableValue<T> extends WatchableValue<T> implements Trackable {
     this.value = this.defaultValue;
   }
 }
+export class LockableValue<T> extends TrackableValue<T> implements Trackable {
```

Comment:
Meant to implement LockableValueInterface here?

Reply:
LockableValueInterface is here https://github.com/seung-lab/neuroglancer/blob/166889f69c8cc049533b8a2023778829c5bce625/src/neuroglancer/trackable_value.ts#L175